Cabal/src/Distribution/Simple/Program/Run.hs

```diff
@@ -168,14 +168,16 @@ getProgramInvocationOutput verbosity inv = do
   when (exitCode /= ExitSuccess) $
     die' verbosity $
       "'" ++ progInvokePath inv ++ "' exited with an error:\n" ++ errors
+  -- dieWithException verbosity $
+  -- GetProgramInvocationException (progInvokePath inv) errors
```

Comment:
Leftover?

Reply:
This one needs a rework in the future because of a windows specific failing CI test, but I will remove it for now. Thanks